internal/handlers/lokistack_create.go

```diff
@@ -29,10 +30,17 @@ func CreateLokiStack(ctx context.Context, req ctrl.Request, k k8s.Client) error
 		return kverrors.Wrap(err, "failed to lookup lokistack", "name", req.NamespacedName)
 	}
 
+	img := os.Getenv("LOKI_IMAGE")
+	if img == "" {
+		ll.Error(nil, "could not find LOKI_IMAGE env variable", "name", req.NamespacedName)
+		return nil
```

Comment:
Can we just default this so that we don't have to Setenv in our tests? It's easier if we _allow_ an override, but default to a known value that works such as the one you added in `os.Setenv`